client/src/main/java/com/ibm/crail/tools/CrailBenchmark.java

```diff
@@ -999,7 +1104,8 @@ public static void main(String[] args) throws Exception {
 		boolean useBuffered = true;
 		
 		String benchmarkTypes = "write|writeAsync|readSequential|readRandom|readSequentialAsync|readMultiStream|"
-				+ "createFile|createFileAsync|createMultiFile|getKey|getFile|getFileAsync|enumerateDir|browseDir|"
+				+ "createFile|createFileAsync|createMultiFile|getKey|getFile|getFileAsync|getMultiFile"
+				+ "getMultiFileAsync|enumerateDir|browseDir|"
```

Comment:
Can we make this more generic. Lot's of Strings duplicated and possible spelling errors.
I would like to see something like a map from string -> benchmark method

Reply:
I agree with this, there should be no need for string duplication.

I'd prefer to do this as a cleanup step and will create an issue.